olp-cpp-sdk-core/include/olp/core/client/HttpResponse.h

```diff
@@ -32,6 +33,11 @@ namespace client {
  */
 class CORE_API HttpResponse {
  public:
+  /**
+   * @brief Type alias for HTTP headers.
+   */
+  using HeadersType = std::vector<std::pair<std::string, std::string>>;
```

Comment:
Would a `map` (or and `unordered_map`) be more helpful and easier to use downstream?

Reply:
Or even better – an `std::multimap` that would reflect the way headers are passed to `OlpClient::CallApi`

Reply:
Apparently, it's a big mess :) 
The `http::NetworkRequest` class has a type `RequestHeadersType` that is a vector.
So I propose to deprecate the `http::NetworkRequest::RequestHeadersType`, add  a common `http:: HeadersType` as `using HeadersType = std:: multimap <std::string, std::string>` instead, and use it consistently.

Reply:
I would prefer we use `std::vector<std::pair<std::string, std::string>>` all the way, as it is handier then a multimap and has a better memory footprint.

Reply:
HeadersType was updated to multimap on Friday. Please take a look, what commit I should leave, previous with vector, or last one with multimap.